### PR TITLE
CodeforScience.ScienceFair version 1.0.6.234

### PR DESCRIPTION
**Pull request by oxygen-dioxide**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/21954)

### CODE REVIEW

File: manifests/c/CodeforScience/ScienceFair/1.0.6.234/CodeforScience.ScienceFair.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.3.0.3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: CodeforScience.ScienceFair
+PackageVersion: 1.0.6.234
```
Comment (ItzLevvie) [inline manifests/c/CodeforScience/ScienceFair/1.0.6.234/CodeforScience.ScienceFair.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 1.0.6
```

File: manifests/c/CodeforScience/ScienceFair/1.0.6.234/CodeforScience.ScienceFair.yaml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 0.3.0.3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
+PackageIdentifier: CodeforScience.ScienceFair
+PackageVersion: 1.0.6.234
```
Comment (ItzLevvie) [inline manifests/c/CodeforScience/ScienceFair/1.0.6.234/CodeforScience.ScienceFair.yaml:5]:
```suggestion
PackageVersion: 1.0.6
```

File: manifests/c/CodeforScience/ScienceFair/1.0.6.234/CodeforScience.ScienceFair.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.3.0.3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: CodeforScience.ScienceFair
+PackageVersion: 1.0.6.234
```
Comment (ItzLevvie) [inline manifests/c/CodeforScience/ScienceFair/1.0.6.234/CodeforScience.ScienceFair.installer.yaml:5]:
```suggestion
PackageVersion: 1.0.6
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
Control Panel is displaying this application as `1.0.6` so you'll have to update the version folder as well.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
image

**Review by OfficialEsco — APPROVED** [review]

**Pull request merged**